Northstar.CustomServers/mod/scripts/vscripts/sh_utility_all.gnut

```diff
@@ -1540,6 +1540,20 @@ array<string> function GetAvailableTitanRefs( entity player )
 	return availableTitanRefs
 }
 
+void function RecalculateHighestTitanFDLevel( entity player )
```

Comment:
Same here although this might be too verbose

```suggestion
/// Gets the highest Titan FD level and stores it in the corresponding persistent var.
/// * `player` - The player entity to perform the action on
void function RecalculateHighestTitanFDLevel( entity player )
```